src/rules/objectLiteralSortKeysRule.ts

```diff
@@ -137,6 +142,10 @@ function walk(ctx: Lint.WalkContext<Options>, checker?: ts.TypeChecker): void {
                     break;
                 case ts.SyntaxKind.ShorthandPropertyAssignment:
                 case ts.SyntaxKind.PropertyAssignment:
+                    if (hasBlankLineBefore(ctx.sourceFile, property)) {
```

Comment:
This can be pretty expensive when checked for each property. I'd prefer to only check this right before adding a failure

Reply:
Done.